### DISCUSSION TITLE
Just a minor question, regarding use of the word 'system'.

### DISCUSSION THREAD

**Discussion by Lalarian** [xCuri0/ReBarUEFI#149] [General]
Hello!
I just found about the project and wondered if it could help my old system (i5-6600K + AMD R9 380) to new heights.

Your decription starts with saying 'A UEFI DXE driver to enable Resizable BAR on **systems** which don't support it officially', so this would refer to the **system** as a whole. But after searching and trying to get a grasp on the topic, I found that it probably won't work for me (not for my **system**, as a whole) because the R9 380 does not support ReBaR/SAM because of it's own BIOS. Is this right?

In case I am right I think it might be good to change the beginning sentence to: 'A UEFI DXE driver to enable Resizable BAR on **CPU+motherboard combinations** which don't support it officially' ps it won't confuse people abuot the possibilities.

Thank you! :)

**Comment by xCuri0** [comment]
I don't know if R9 380 supports rebar or not but it's very easy to check using GPU-Z

It's hard to find anyone still using AMD GPUs from before Polaris so none have been reported in the working list.

Btw even if your R9 doesn't support rebar it should still be theoretically possible to enable it using VBIOS modification or alternate method to alter BAR size, this was done for NVIDIA Turing GPUs

**Reply by Lalarian** [reply]
I have checked this and this graphcis crad is not eBar capable.
But this is not what I was talking about. I just wanted to say the word 'system' in the README should be swapped with 'CPU+mainboard combination', because the word system is missleading, possibly making people think they could for sure mod their system, despite this not being possible because of the graphics card not being able to :)
